### ISSUE TITLE
Update peer dependency version

### ISSUE THREAD

**Issue by RussCoder** [phosphor-icons/vue#9]
Hello! Thank you for the nice icons.

I have switched to npm 7 and it installs peer dependencies by default. So when I updated Vue to `3.0.11` I got a conflict with `phosphor-vue`'s peer dependencies:

```
  "peerDependencies": {
    "vue": "^3.0.0 <3.0.3"
  },
  ```

Could you, please, change it to `^3.0.0` and make a new patch release?
Right now I can run `npm install` only with `--legacy-peer-deps` (when there is no `package-lock.json`).

**Comment by Clex1o1** [comment]
Are there any plans to add vue 3 support?

**Comment by rektdeckard** [comment]
Vue 3 is supported, but due to a bundler bug it's not currently
tree-shakable. Check the vue3 branch for info, and `npm install
***@***.***` to install. We'll update the peer dependency to be
friendly to recent versions.

**Comment by raphaelmenges** [comment]
I am facing the same issue and since upgrading to the latest npm version I need to install my dependencies using `npm install --force` or at least `npm install --legacy-peer-deps`: <https://stackoverflow.com/questions/66020820/npm-when-to-use-force-and-legacy-peer-deps>

Otherwise following error occurs:
```
npm ERR! Could not resolve dependency:
npm ERR! peer vue@"^3.0.0 <3.0.3" from phosphor-vue@4.4.0
npm ERR! node_modules/phosphor-vue
npm ERR!   dev phosphor-vue@"4.4.0" from the root project
```

Moreover, I am puzzled that the dependency of phosphor-vue to vue is listed in the devDependencies with `"vue": "^3.2.26"` and in peerDependencies with `"vue": "^3.0.0 <3.0.3"`. It does not match up!

**Comment by Lootjs** [comment]
@rektdeckard , @dnlsndr can you guys remove this restriction or accept my PR?

**Comment by dnlsndr** [comment]
I've merged the PR, @rektdeckard can you please deploy a new npm version?

**Comment by raphaelmenges** [comment]
Any updates on this? Using `--legacy-peer-deps` all the time feels quite award 🥲

**Comment by jordanshatford** [comment]
Is there any update to this being released @rektdeckard @dnlsndr

**Comment by martinleo** [comment]
Also waiting for this 😓

**Comment by dnlsndr** [comment]
As many people have requested this now, and Tobias seems to be AFK for the moment, I'll see to deploy a temporary npm package that you can use in the mean time

**Comment by dnlsndr** [comment]
Hi there, I just wanted to let you know that I am currently having a look into how we can fix the deployment pipeline for this repo. In the meantime, feel free to use my reimplementation of the phosphor-vue library for Vue 3 [here](https://github.com/dnlsndr/vue-phosphor-icons).

Since my version holds a lot of changes compared to the phosphor-vue library, I didn't want to break anything by commiting these changes to mainline. Thus I created my own.
Note: My library does **NOT** have a global install for your vue application. This means you can't just put in a app.use() in your main.ts. You'll need to import each icon as soon as you want to use it. The reasoning behind this, is that tree-shakers still often get confused with the icon rollup builds, and often just import the whole icon library at once if you use even one icon. This again bloats your application and increases page load-times n-fold. It's anyways an antipattern generally advised against.

**Comment by teseu** [comment]
> Hi there, I just wanted to let you know that I am currently having a look into how we can fix the deployment pipeline for this repo. In the meantime, feel free to use my reimplementation of the phosphor-vue library for Vue 3 [here](https://github.com/dnlsndr/vue-phosphor-icons).
> 
> Since my version holds a lot of changes compared to the phosphor-vue library, I didn't want to break anything by commiting these changes to mainline. Thus I created my own. Note: My library does **NOT** have a global install for your vue application. This means you can't just put in a app.use() in your main.ts. You'll need to import each icon as soon as you want to use it. The reasoning behind this, is that tree-shakers still often get confused with the icon rollup builds, and often just import the whole icon library at once if you use even one icon. This again bloats your application and increases page load-times n-fold. It's anyways an antipattern generally advised against.

I liked your reimplementation just fine. It's better then use workarrounds

**Comment by rektdeckard** [comment]
Apologies to everyone here for leaving this project unmaintained and issues ignored. After a long, unintentional hiatus, I'm back at work on phosphor, and we have some plans for a new major release soon, across all libs and implementations:

- More than 200 new icons, nearly complete as of writing this
- Shipping assets and components using minified paths instead of basic shapes for ~30% bundle size reduction
- Migrating to modern build tools (Vite)
- Streamlining the update and deployment by using [@phosphor-icons/core](https://github.com/phosphor-icons/core) as source of truth for assets
- Better tree-shaking support in places that need it (I.E. here)
- Moving all libs under the @phosphor-icons org / namespace on NPM

I understand if many of you have moved permanently to [@dnlsndr/vue-phosphor-icons](https://github.com/dnlsndr/vue-phosphor-icons), and I don't blame you! I'd love to upstream some of those improvements here as part of this work. But I wanted to let you know that I'm planning to revive this repo. 

Have said it several times before here, but I'm not a Vue dev -- so I'm looking to appoint someone (@dnlsndr ?) to have write access to our NPM org, as well as permissions here, to keep things running. In the mean time, I have published the peer dependency fix in #25 under `phosphor-vue@4.4.1` / `phosphor-vue@next`.

Thanks everyone for the patience
